### PR TITLE
Pro 4 email sending survey

### PR DESCRIPTION
**Pull request by GulzatB**

### CODE REVIEW

**Comment by GulzatB** [comment]
Kindly ask you to look through my code, pls.

File: force-app/main/default/classes/CaseService.cls [hunk 1]
```diff
@@ -1,4 +1,17 @@
...
+     * Method Implements Email sending feedback-survey logic on Case
+     * This Email contains button, which will redirect to Google form. 
+     * 
+     * @param newCase new inserted cases
+     * @author gulzatbaimyrzaeva 
+     */
+    public static void sendEmailSurvey(List<Case> newCases){
+        for(Case singleCase : newCases){
+            if(singleCase.Status == 'Closed') {
+                EmailService.send(singleCase.Status);
```
Comment (belekomurzakov) [inline force-app/main/default/classes/CaseService.cls]: pls push ur new changes with singleCase.SuppliedEmail

**Review by belekomurzakov — CHANGES_REQUESTED** [review]

File: force-app/main/default/classes/CaseService.cls (hunk 1 above)
Comment (GulzatB) [reply force-app/main/default/classes/CaseService.cls]: I've pushed.

**Review by belekomurzakov — APPROVED** [review]
Looks great, pls merge

**Comment by GulzatB** [comment]
How about @test?

**Pull request merged**